### DISCUSSION TITLE
Make website's viewport scroolbar dark

### DISCUSSION THREAD

**Discussion by reuixiy** [remix-run/remix#675] [General]
Hi there,

I'm reading Remix Docs and found the scroolbar is so bright...

This can be easily fixed by adding `color-scheme` HTML meta tag or CSS property, see https://web.dev/color-scheme/

Before

![default](https://user-images.githubusercontent.com/28993043/143557184-ff508db1-7a1d-4f3a-baa6-be86dfa1a9d0.png)

After
![dark](https://user-images.githubusercontent.com/28993043/143557209-14ba4900-4af4-4610-9ff4-42a7764d575b.png)

P.S. I can't find website's repository and issues seems not a correct place so I just put this feedback here...

**Comment by matmilbury** [comment]
@kentcdodds we're waiting for the website repo to be open sourced too. A simple PR in this case would be ideal...
